dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsDisplayDataModel.ts

```diff
@@ -603,31 +622,31 @@ export const euTaxonomyForNonFinancialsDisplayDataModel = [
         ],
       },
       {
-        name: "totalEnablingShare",
-        label: "Total Enabling CapEx",
+        name: "enablingShare",
+        label: "Enabling CapEx",
         fields: [
           {
-            name: "totalEnablingShare",
-            label: "Total Enabling CapEx",
+            name: "enablingShareInPercent",
+            label: "Enabling CapEx (%)",
             description: "",
             unit: "",
-            component: "UndefinedFormField",
+            component: "PercentageFormField",
             evidenceDesired: false,
             required: false,
             showIf: (): boolean => true,
           },
         ],
       },
       {
-        name: "totalTransitionalShare",
```

Comment:
Why is this different from the totalTransitionalShare for Opex?

Reply:
Is this still an open question? In my version of the `EuTaxonomyForNonFirnancialsDisplayDataModel.ts` there is no `totalTransitionalShare`, neither in CapEx nor in OpEx

Reply:
Yes it is gone now. For me too.

Reply:
I got confused by the github display. Closing this